sources/packages/web/src/views/aest/SupportingUser.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <header-navigator
+    title="Back to student applications"
+    :routeLocation="{
+      name: AESTRoutesConst.STUDENT_APPLICATIONS,
+      params: { studentId },
+    }"
+    subTitle="Financial Aid Application"
+  >
+  </header-navigator>
+  <full-page-container class="my-2">
+    <formio
+      v-if="formName"
+      :formName="formName"
+      :data="formData"
+      :readOnly="true"
+    ></formio>
+    <Message v-else severity="warn" :closable="false">
+      <strong>
+        Supporting User has not submitted the application.
+      </strong>
+    </Message>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import formio from "@/components/generic/formio.vue";
+import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
+import { ref, onMounted } from "vue";
+import { SupportingUsersService } from "@/services/SupportingUserService";
+import { useFormatters } from "@/composables";
+import HeaderNavigator from "@/components/generic/HeaderNavigator.vue";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+
+export default {
+  components: {
+    formio,
+    FullPageContainer,
+    HeaderNavigator,
+  },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+    supportingUserId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const { dateOnlyLongString } = useFormatters();
+    const formName = ref();
+    const formData = ref();
+
+    onMounted(async () => {
+      const supportingUsersData = await SupportingUsersService.shared.getSupportingUserData(
+        props.supportingUserId,
+      );
+      formName.value = supportingUsersData.formName;
+      let contactAddress = {};
+      // Here there is only one address for now
+      if (supportingUsersData.contactInfo.addresses.length === 1) {
+        contactAddress = {
+          city: supportingUsersData.contactInfo.addresses[0].city,
```

Comment:
Please avoid using the same indexer over and over again.
Please assign this to one variable: _supportingUsersData.contactInfo.addresses[0]_